src/components/map-container.js

```diff
@@ -173,86 +259,8 @@ export default function MapContainerFactory(MapPopover, MapControl) {
     };
 
     /* component render functions */
-    /* eslint-disable complexity */
-    _renderMapPopover() {
```

Comment:
i converted this method into a react component to take advantage of react render optimization